src/app/lib/utilities/preprocessor/rules/injectAds.js

```diff
@@ -0,0 +1,56 @@
+import compose from 'ramda/src/compose';
+import {
+  getJsonContent,
+  getBlocks,
+  mergeJsonRawWithBlocks,
+} from './addIdsToBlocks';
+
+const adBlock = {
```

Comment:
For context this is just a blank ad block for now. Once we know more about how ads will work and what ad config we need we can add more attributes to this new block.